jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/internal/HttpChannelState.java

```diff
@@ -452,7 +453,7 @@ else if (ExceptionUtil.areNotAssociated(_failure.getFailure(), x) && _failure.ge
                 _onContentAvailable = null;
 
                 // If a write call is in progress, take the writeCallback to fail below
-                Runnable invokeWriteFailure = _response.lockedFailWrite(x);
+//                Runnable invokeWriteFailure = _response.lockedFailWrite(x);
```

Comment:
@sbordet So if we don't inform a write of failure here, then what is the process for failing a write that is in process.
I.e. If a handler is trying to write a large response and is write blocked, then the connection is reset, then are we sure that the write to the HttpStream will be failed?   I think the stream contract is such that it will be failed and this method is really only for failures when there is no IO operation, or idleTimeouts or application Content.Source.fail(Throwable) calls.

Reply:
probably need to rename the method if it stays this way.
Note I'm a bit wrong about idletimeouts, as they are first reported the the onIdleTimeout method, which still does fail the write side and only comes here if there are no IO operations.